library_generation/generate_library.sh

```diff
@@ -107,9 +107,20 @@ mkdir -p "${output_folder}/${destination_path}"
 # get a fixed order.
 folder_name=$(extract_folder_name "${destination_path}")
 pushd "${output_folder}"
-proto_files=$(find "${proto_path}" -type f  -name "*.proto" | sort)
+find_depth=""
+case "${proto_path}" in
+  "google/api" | "google/cloud" | "google/iam/v1" | "google/rpc")
+    find_depth="-maxdepth 1"
+    ;;
+esac
+proto_files=$(find "${proto_path}" ${find_depth} -type f  -name "*.proto" | sort)
 # include or exclude certain protos in grpc plugin and gapic generator java.
 case "${proto_path}" in
+  "google/cloud")
```

Comment:
Is this folder generated into java-common-protos [here](https://github.com/googleapis/sdk-platform-java/blob/main/java-common-protos/proto-google-common-protos/src/main/proto/google/cloud/extended_operations.proto) eventually?

Reply:
Yes.